js/services/MigrationService.js

```diff
@@ -30,7 +30,7 @@ DuckieTV.run(['SettingsService', function(SettingsService) {
   // update quality list
   if (!localStorage.getItem('1.1.6updateQualityList')) {
```

Comment:
because the key 1.1.6updateQualityList has already been used before , it will mean that your update to refresh the list for existing users will not take place.
Suggest you replace all instances' of 1.1.6updateQualityList with 1.1.6updateQualityList2 or whatever to make the key unique.

Reply:
Ah kk, wasn't sure if that was just for 1.1.5 users migrating over. Will add now.